src/lsg/patterns/icons/index.tsx

```diff
@@ -41,6 +42,7 @@ const icons: { readonly [key: string]: JSX.Element[][] | JSX.Element[] } = {
 		[<path key="arrow" d="M17.5 12l-8.486 8.485L7.6 19.071 14.671 12 7.6 4.929l1.414-1.414z" />]
 	],
 	[IconName.ArrowFill]: [[<path key="arrowFill" d="M8 4l8 8-8 8z" />]],
```

Comment:
please refactor this one the `ArrowFillRight`with that it will be easier to use the correct one

Reply:
✅